src/functions-reference/transform_functions.qmd

```diff
@@ -0,0 +1,537 @@
+sum to `0`.
+
+The returned vector will have one extra element compared to the input `y`.
+
+This function also increments the `target` variable
+with the corresponding change of variables adjustment.
+{{< since 2.37 >}}
+
+<!-- vectors; sum_to_zero_unconstrain; (vectors x); -->
+\index{{\tt \bfseries sum\_to\_zero\_unconstrain }!{\tt (vectors x): vectors}|hyperpage}
+
+`vectors` **`sum_to_zero_unconstrain`**`(vectors x)`<br>\newline
+Takes a vector `x` with elements that sum to `0` and returns the
+corresponding free vector.
+
+This returned vector will have one fewer elements compared to the input `x`.
+{{< since 2.37 >}}
+
+### Unit vectors
+
+These functions perform the mathematical transform described in the
+[Unit Vector section](https://mc-stan.org/docs/reference-manual/transforms.html#unit-vector.section).
+
+<!-- vectors; unit_vectors_constrain; (vectors y); -->
+\index{{\tt \bfseries unit\_vectors\_constrain }!{\tt (vectors y): vectors}|hyperpage}
+
+`vectors` **`unit_vectors_constrain`**`(vectors y)`<br>\newline
+Takes a free vector `y` and returns a vector with Euclidian length `1`, e.g.
+`norm2(unit_vectors_constrain(y)) == 1` for any `y` that has a positive
+and finite norm itself. Note that, in particular, this function rejects
+if given a vector of all zeros.
+{{< since 2.37 >}}
+
+<!-- vectors; unit_vectors_jacobian; (vectors y); -->
+\index{{\tt \bfseries unit\_vectors\_jacobian }!{\tt (vectors y): vectors}|hyperpage}
+
+`vectors` **`unit_vectors_jacobian`**`(vectors y)`<br>\newline
+Takes a free vector `y` and returns a vector with Euclidian length `1`.
+This function rejects if given a vector of all zeros.
+
+This function also increments the `target` variable
+with the corresponding change of variables adjustment.
+{{< since 2.37 >}}
+
+<!-- vectors; unit_vectors_unconstrain; (vectors x); -->
+\index{{\tt \bfseries unit\_vectors\_unconstrain }!{\tt (vectors x): vectors}|hyperpage}
+
+`vectors` **`unit_vectors_unconstrain`**`(vectors x)`<br>\newline
+Takes a vector `x` of unit length and returns the corresponding free vector.
+{{< since 2.37 >}}
+
+## Transforms for constrained matrices
+
+Similarly to the above, `vectors` means a `vector` or array thereof,
+and `matrices` means a `matrix` or array thereof.
+
+### Cholesky factors of correlation matrices
+
+
+These functions perform the mathematical transform described in the
+[Cholesky Factors of Correlation Matrices section](https://mc-stan.org/docs/reference-manual/transforms.html#cholesky-factors-of-correlation-matrices).
+
+<!-- matrices; cholesky_factor_corr_constrain; (vectors y, int K); -->
+\index{{\tt \bfseries cholesky\_factor\_corr\_constrain }!{\tt (vectors y, int K): matrices}|hyperpage}
+
+`matrices` **`cholesky_factor_corr_constrain`**`(vectors y, int K)`<br>\newline
+Takes a vector `y` and integer `K`, where `length(y) == choose(K, 2)`, and returns
+a `K` by `K` Cholesky factor of a correlation matrix.
```

Comment:
You can mention that this is a Cholesky factor of a covariance matrix (i.e., lower triangular with strictly positive diagonal) but with the added constraint that each row is of unit length so that the resulting product has a unit diagonal.